packages/firebase_vertexai/firebase_vertexai/lib/src/live_api.dart

```diff
@@ -0,0 +1,302 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import 'api.dart';
+import 'content.dart';
+import 'error.dart';
+
+/// Represents the available voice options for speech synthesis.
+enum Voices {
+  // ignore: public_member_api_docs, constant_identifier_names
+  Aoede('Aoede'),
+
+  // ignore: public_member_api_docs, constant_identifier_names
+  Charon('Charon'),
+
+  // ignore: public_member_api_docs, constant_identifier_names
+  Fenrir('Fenrir'),
+
+  // ignore: public_member_api_docs, constant_identifier_names
+  Kore('Kore'),
+
+  // ignore: public_member_api_docs, constant_identifier_names
+  Puck('Puck');
+
+  const Voices(this._jsonString);
+  final String _jsonString;
+
+  // ignore: public_member_api_docs
+  String toJson() => _jsonString;
+}
+
+/// Configures speech synthesis settings.
+class SpeechConfig {
+  /// Creates a [SpeechConfig] instance.
+  ///
+  /// [voice] (optional): The desired voice for speech synthesis.
+  SpeechConfig({this.voice});
+
+  /// The voice to use for speech synthesis.
+  final Voices? voice;
+  // ignore: public_member_api_docs
+  Map<String, Object?> toJson() => {
+        if (voice != null)
+          'voice_config': {
+            'prebuilt_voice_config': {'voice_name': voice!.toJson()}
+          }
+      }; // Or null
```

Comment:
What does this mean? Did it intend to follow the conditional and was put here because it follows a closing brace?

Reply:
Deleted the comment. Put it there before to remind myself it will be empty in the json if no voice set.